src/main/resources/edu/cornell/kfs/fp/document/datadictionary/BudgetAdjustmentDocument.xml

```diff
@@ -10,4 +10,19 @@
     <property name="documentClass" value="edu.cornell.kfs.fp.document.CuBudgetAdjustmentDocument"/>
     <property name="documentPresentationControllerClass" value="edu.cornell.kfs.fp.document.authorization.CuBudgetAdjustmentDocumentPresentationController"/>
    </bean>
+   
+   <bean id="BudgetAdjustmentDocument-workflowAttributes"
+          parent="BudgetAdjustmentDocument-workflowAttributes-parentBean">
+        <property name="routingTypeDefinitions">
+            <map>
+                <entry key="Account" value-ref="RoutingType-AccountingDocument-Account"/>
+                <entry key="AccountingOrganizationHierarchy"
+                       value-ref="RoutingType-AccountingDocument-OrganizationHierarchy"/>
+                <entry key="ObjectCode" value-ref="RoutingType-AccountingDocument-ObjectCode"/>
+                <entry key="Fund" value-ref="RoutingType-AccountingDocument-Fund"/>
```

Comment:
Can we use one of the field override tags in workflow attributes?  It might make some of these changes simpler

Reply:
There is a FieldOverrideForListElementInsert defined for lists but I don't see one for a map

Reply:
Thats unfortunate, thanks for looking.